Book-Store-MVC/Controllers/BookController.cs

```diff
@@ -14,84 +14,47 @@ namespace Book_Store_MVC.Controllers
 {
     public class BookController : Controller
     {
-        private readonly IGenericRepository<Book> genericRepository;
+        private readonly BookRepository  genericRepository;
```

Comment:
why a BookRepo named Genericrepo ?
why !!!!!!!!!!!!!!!

Reply:
sorry i forgot to change it